x-pack/platform/test/alerting_api_integration/security_and_spaces/group3/tests/maintenance_window/internal/find_maintenance_windows.ts

```diff
@@ -154,58 +154,6 @@ export default function findMaintenanceWindowTests({ getService }: FtrProviderCo
           }
         });
 
-        it('throw an error for find maintenance window request with pagination if docs count more 10k', async () => {
```

Comment:
This should still work, right?

Reply:
yes but the schema does not allow `page` or `per_page` values to be > 100, so we can never reach more than 100 * 100 :D